### PR TITLE
Added Home Facility name beside doctors name while assigning doctor in Consutation Page

### LINKED ISSUE #3862: Different Doctors with same name can't be differentiated in telemedicine
**Describe the bug**

Currently, Different Doctors with same name can't be differentiated in telemedicine section of consultation creation page.

image

**Solution**

Add Home Facility of the doctor along with their name

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes
Fixes #3862 

Added Home Facility name beside doctors name while assigning doctor in Consutation Page
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
test approved

File: src/Components/Users/models.tsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
+interface home_facilty_object {
```
Comment (khavinshankar) [inline src/Components/Users/models.tsx]:
```suggestion
interface HomeFacilityObjectModel {
```

File: src/Components/Users/models.tsx [hunk 2]
```diff
@@ -15,4 +19,5 @@ export interface UserModel {
   is_superuser?: boolean;
   verified?: boolean;
   last_login?: Date;
+  home_facility_object?: home_facilty_object;
```
Comment (khavinshankar) [inline src/Components/Users/models.tsx]:
```suggestion
  home_facility_object?: HomeFacilityObjectModel;
```

**Review by khavinshankar — CHANGES_REQUESTED** [review]
just couple refactors

**Review by rithviknishad — APPROVED** [review]
@khavinshankar @nihal467 @patelaryan7751 how about showing the associated facility without "@" prefix but in `text-gray-700` instead of having "@ facility name"?

**Comment by khavinshankar** [comment]
> @khavinshankar @nihal467 @patelaryan7751 how about showing the associated facility without "@" prefix but in `text-gray-700` instead of having "@ facility name"?

that's a nice idea

File: src/Components/Common/OnlineUsersSelect.tsx [hunk 3]
```diff
@@ -199,7 +199,12 @@ export const OnlineUsersSelect = (props: Props) => {
                             }
                           ></span>
                           <span className="font-normal block truncate">
-                            {user.first_name} {user.last_name}
+                            {user.first_name} {user.last_name}{" "}
+                            {`${
+                              user.home_facility_object?.name
+                                ? `@ ${user.home_facility_object?.name}`
+                                : " "
+                            }`}
```
Comment (khavinshankar) [inline src/Components/Common/OnlineUsersSelect.tsx]:
```suggestion
                            {user.home_facility_object?.name && 
                                 <span className="text-gray-700 ml-2">{user.home_facility_object?.name}</span>
                            }
```

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@nihal467 can you retest this pr

image

greyed out the facility name instead of @ facility name

**Comment by nihal467** [comment]
test approved

**Pull request merged**